src/db/migrations/20240408161125_add_monthly_report_column.js

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export function up (knex) {
+  return knex.schema.table('subscribers', table => {
+    table.dropColumn("monthly_email_optout")
+    table.dropColumn("monthly_email_at")
```

Comment:
These drops might take a while to run.

Are we sure that they don't need to be migrated at all, are there existing accounts that have opted-out that will be surprised to receive this? Or are we already accounting for that because we are restricting the date that the email is sent?

Reply:
I'm pretty sure these are from the old PM who's no longer here. I can ask Tony about it

Reply:
Also good thinking about the scale concern re `DROP COLUMN`. 

See the postgres `alter table` doc:
```
The DROP COLUMN form does not physically remove the column, but simply makes it invisible to SQL operations. Subsequent insert and update operations in the table will store a null value for the column. Thus, dropping a column is quick but it will not immediately reduce the on-disk size of your table, as the space occupied by the dropped column is not reclaimed. The space will be reclaimed over time as existing rows are updated.
```

So I don't think it's an issue in practice